app/helpers/sessions_helper.rb

```diff
@@ -3,16 +3,36 @@ def log_in(user)
     session[:user_id] = user.id
   end
 
+  def remember(user)
+    user.remember
+    cookies.permanent.signed[:user_id] = user.id
+    cookies.permanent[:remember_token] = user.remember_token
+  end
+
   def current_user
-    return unless session[:user_id]
-    @current_user ||= User.find_by id: session[:user_id]
+    if (user_id = session[:user_id])
```

Comment:
- xóa ()
- Tại sao ở đây lại là phép gán mà k phải là 1 toán tử so sánh

Reply:
thưa anh bởi vì sau phép gán này là 1 câu lệnh if có điều kiện rồi anh !